store/src/java/com/zimbra/cs/filter/ZimbraMailAdapter.java

```diff
@@ -572,7 +616,13 @@ private boolean isPathContainedInFiledIntoPaths(String folderPath) {
     private void fileInto(String folderPath)
     throws ServiceException {
         folderPath = CharMatcher.is('/').trimFrom(folderPath); // trim leading and trailing '/'
-        if (ZimbraLog.filter.isDebugEnabled()) {
+        if (LC.lmtp_extended_logs_enabled.booleanValue()) {
+            String messageId = Mime.getMessageID(handler.getMimeMessage());
+            String sender = Mime.getSender(handler.getMimeMessage());
+            ZimbraLog.filter.info(
+                    appendFlagTagActionsInfo("fileinto " + folderPath, getFlagActions(), getTagActions())
```

Comment:
Like Yogesh said please create a method for evaluating the sender.

Reply:
done